lib/nodekey/NodeKey.ts

```diff
@@ -0,0 +1,86 @@
+import CryptoJS from 'crypto-js';
+import { randomBytes } from 'crypto';
+import secp256k1 from 'secp256k1';
```

Comment:
What about creating a `crypto` layer/folder, to wrap up all usage in external libs?

Reply:
I'm not sure how this would be structured. All these modules are from third party dependencies outside this code module, would the `crypto` layer just import these and re-export them? I'm not sure where we'd be using this layer either. My preference would be to leave this alone for now until we have a better idea of how we're using cryptography outside of `NodeKey` and ecdsa sign/verify.

Reply:
That's fine, we can leave this out for now.